src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -109,7 +105,11 @@ export const SignOrExecuteForm = ({
           decodedDataLoading={decodedDataLoading}
           showMultisend={!props.isBatch}
           showToBlock={props.showToBlock}
-        />
+        >
+          <ErrorBoundary fallback={<div>Error parsing data</div>}>
```

Comment:
why is this now inside of the decoded TX?

Reply:
To place it under the swap into but above multisend details like it was for other tx types.
Alternatively we move it all the way down

Reply:
Yeah its a bit surprising. Also with rendering the children if the decoded data does not exist.
I would not block it and will still approve but I think at some point we should maybe let decodedData only decode the transaction and move the SwapOrder details outside of it and render it separately. Then we have smaller components. I am also not sure if we need all those conditions within DecodedTx.
We suppress the `SendToBlock` for instance just to render our own `SendToBlock` (which looks identical) within the Swap order.

Reply:
Agree! I didn’t want to do a big refactoring in this scope but we can do it as a follow up.